pkg/minishift/addon/addon_meta.go

```diff
@@ -99,3 +115,10 @@ func (meta *DefaultAddOnMeta) splitAndTrim(s string) []string {
 	}
 	return cleanSplit
 }
+
+func checkVersionSemantic(version string) bool {
+	// Strict match for <major> or <major>.<minor> or <major>.<minor>.<patch>
+	// (>=|>|<|<=)3.6.0, (>=|>|<|<=)3.6.0
+	match, _ := regexp.MatchString("^(|>|>=|<|<=)[0-9](.[0-9]){0,2}(|\\s*,\\s*(|>|>=|<|<=)[0-9](.[0-9]){0,2})$", version)
```

Comment:
So does this not only work for single digit versions, eg 3.6.0? You only match a digit one [0-9]. What if we start getting 3.10.0 or 3.7.10 or something like this? Should it not be [0-9]+?

Reply:
@hferentschik right, I somehow considered only one digit :(